server/live-loan-router.js

```diff
@@ -0,0 +1,150 @@
+const get = require('lodash/get');
+const express = require('express');
+const argv = require('./util/argv');
+const config = require('../config/selectConfig')(argv.config);
+const fetch = require('./util/fetch');
+
+const drawLoanCard = require('./util/live-loan/live-loan-draw');
+
+function getLoansFromCache(loginId, cache) {
+	return new Promise(resolve => {
+		cache.get(`recommendations-by-login-id-${loginId}`, (error, data) => {
+			let parsedData = [];
+			if (error) {
+				console.error(JSON.stringify({
+					meta: {},
+					level: 'error',
+					message: `MemJS Error Getting recommendations-by-login-id-${loginId}, Error: ${error}`
+				}));
+			}
+			if (data) {
+				parsedData = JSON.parse(data);
+			}
+			resolve(parsedData);
+		});
+	});
+}
+
+function setLoansToCache(loginId, loans, cache) {
+	return new Promise((resolve, reject) => {
+		const cacheTime = 30 * 60; // 30 minutes
```

Comment:
@emuvente In my testing I tried setting this to 1 minute and did some additional logging. I was still getting cached results several minutes later. Ever tinkered with expiration times?

Reply:
it looks like you're supposed to pass the time as the expires property in an object for the third argument of `set()`, rather than just the time https://github.com/memcachier/memjs#settings-values

Reply:
ahh, thank you!